### PR TITLE
Use process.defaultApp to detect built electron app

### PR DESCRIPTION
**Pull request by mleguen**
If process.defaultApp is false (for built electron app), remove only the first element of process.argv (the app binary) and not the two first elements (node + the app script).

See https://github.com/electron/electron/issues/4690#issuecomment-217435222 for mor info.

Solve part of #685

### CODE REVIEW

File: lib/process-argv.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function getProcessArgvBinIndex () {
+  // Built Electron app: app argv1 argv2 ... argvn
+  // (see https://github.com/electron/electron/issues/4690#issuecomment-217435222)
+  if (process.defaultApp === false) return 0
```
Comment (bcoe) [inline lib/process-argv.js]: is `process.defaultApp` set by electron, probably worth noting this in the comment as well.

Comment (mleguen) [reply lib/process-argv.js]: OK, I'll do it ASAP

Comment (mleguen) [reply lib/process-argv.js]: Comment added

**Comment by bcoe** [comment]
@mleguen friendly reminder to make sure you use conventional commits when you merge, or else stuff doesn't show up in the next release, I went ahead and amended.

**Comment by mleguen** [comment]
@bcoe Oops! Sorry and thanks. Would you have a link to some doc about the requested commit syntax (is it the same as the one used by nest.js?) for me to do it as expected next time?

**Pull request merged**